learning-pathways/train-the-trainers.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: learning-pathway  # (uncomment this line to activate it)
+
+title: Train the Trainers
+description: |
+  This pathway introduces trainers to learning principles, training techniques, lesson, session, course, and material design as well as assessment and feedback. This is has been developed for by trainers in the bioinformatics but is suitable for all trainers and educators in higher education.
+tags: [training, trainers]
+
+pathway:
+  - section: "Module 1: Learning principles"
+    description: |
+        This module covers the following questions:
+        - What are the main principles that drive learning? 
+        - How can these principles be applied to training and teaching?
+
+        At the end of this module, learners should be able to:
+        - Describe how learning works according to a few learning models
+        - Explain how learning models can help you improve your teaching in the classroom
+        - List learning strategies and principles suggested by evidence-based research results
+
+    tutorials:
+      - name: learning-principles
+        topic: contributing
+
+  - section: "Module 2: Design and plan session, course, materials"
+    description: |
+        This module covers the following questions:
+        - What is the structured approach to course design? 
+        - How to articulate learning outcomes commensurate with the cognitive complexity of the target learning?
+        - How to devise learning experiences and course content?
+        - What are the steps that can help trainers devise and deploy effective courses?
+
+        At the end of this module, learners should be able to:
+        - List five key phases of curriculum & course development
+        - Explain the primary role of learning outcomes
+        - Write learning outcomes for a course
+        - Identify the Bloom's-level accomplishments that different types of learning experience are likely to support
+        - Describe the role of learning outcomes in selecting relevant content
+        - Distinguish different types of assessment & their role in supporting learner progression towards learning outcomes
+        - Summarise the benefits of course evaluation
+
+    tutorials:
+      - name: contributing
+        topic: design
+
+
+      - section: "Module 3: Training techniques to enhance learner participation and engagement"
```

Comment:
the indentation is off on this line so module 3 isn't showing up: https://training.galaxyproject.org/training-material/learning-pathways/train-the-trainers.html